app/styles/_library/_objects.sections.scss

```diff
@@ -369,13 +379,20 @@
 .c-basic__header {
   padding: var(--space-3) 0;
   position: relative;
-  @include typeface(header, 16);
-
-  @include media("<=medium") {
-    @include typeface(header, 14);
+  @include typeface(header, 14);
+  @include media(">=medium") {
+    @include typeface(header, 16);
+  }
+}
+.c-basic__heading {
+  @include typeface(header, 14);
+  @include media(">=medium") {
+    @include typeface(header, 16);
   }
```

Comment:
At first i thought this was a typo, but i looked at the client code and we really do have a `c-basic__header` AND `c-basic__heading`. I know they're different html elements and we want `flat page` styles for all the elements so it makes sense, but I was still surprised.

I wonder if we need separate flat page styles AND base styles. 🤔

Reply:
Yeah that tripped me up for a second. Probably a good idea, food for thought.